Pipfile

```diff
@@ -0,0 +1,21 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+websockets = "==9.1"
+asyncio = ">=3.4.3"
+rx = "==3.1.1"
+deriv-api = {editable = true, path = "."}
+
+[dev-packages]
+pytest = "*"
+pytest-runner = "*"
+pytest-mock = "*"
+pytest-asyncio = "*"
+pdoc3 = "*"
+coverage = "===4.5.4"
+
+[requires]
+python_version = "3.9.6"
```

Comment:
Can we add a comment to explain why we need 3.9.6 ?
like `there is a bug in 'websockets' package when python is 3.9.7.  Hope it will be fixed in 3.9.8`

Reply:
Added a note in README